scanreport.sh

```diff
@@ -1,21 +1,24 @@
 #!/bin/bash
 
 function usage {
-	echo "usage: $1 [-f nmap.grepable] [-i IP] [-p port] [-s service] [-P protocol]"
+	echo "Usage: $1 [-f nmap.grepable] [-i IP] [-p port] [-s service] [-P protocol] [-a]"
+	echo "  -a: outputs matching IP addresses only"
 }
 
 db=""
 ip=""
 port=""
 all=0
 proto=""
-while getopts "f:i:p:P:s:" OPT; do
+addressonly=""
+while getopts "f:i:p:P:s:a" OPT; do
```

Comment:
If -a needs an argument we can just use this:

`while getopts "f:i:p:P:s:a:" OPT; do`